### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-263.1][WIP] For an element only 1 region must be marked.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11245]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
For an element only 1 region must be marked.
When we develop UI using NGX component. In VPC mode, when navigated in R key, it is observed that for the all UI pages multiple regions like 'Example dynamic page' and 'Banner', 'breadcrumb' are mentioned.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Accessibility expert can help

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide more information?

**Comment by JyotiKumari1507** [comment]
During Accessibility testing , team found for Dynamic page region screen reader is reading as  'Example dynamic page' and 'Banner', 'breadcrumb' . 'Example dynamic page' we are removing from ariba-label from my side. Other we are not sure. We will have more discussion on this next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by JyotiKumari1507** [comment]
@droshev  Attaching screen reader screenshot for more details:
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/09fd1905-0048-4c86-9693-630ff89e93bf">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise us here? thanks

**Comment by droshev** [comment]
We will discuss it internally and try to understand more

**Comment by mikerodonnell89** [comment]
@JyotiKumari1507 It appears your screenshots are from an internal application so we're unable to replicate the issue to test and fix on our end. And without additional specific details (i.e. example code, example stackblitz, code from our library etc) we're unable to move forward with this issue. I will close this but please open another issue with more details if this issue is still relevant
